zipkin-server/src/test/java/zipkin2/server/internal/throttle/FakeCall.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package zipkin2.server.internal.throttle;
+
+import java.io.IOException;
+import java.util.concurrent.RejectedExecutionException;
+import zipkin2.Call;
+import zipkin2.Callback;
+
+class FakeCall extends Call<Void> {
```

Comment:
probably extending Call.Base where possible is a good idea. Also inline types like this unless used in multiple classes

Reply:
I waffled a bit on extending Call.Base a bit in ThrottledCall.  Should I do it there?  I can probably get rid of some logic around `canceled` if I do.

As for inlining the type, how would I do that with a call?  It's not a FunctionalInterface.  It is also used a few tests in ThrottledCallTest.   Is that sufficient to keep this separate?

Reply:
sorry inling was a poor choice of words. I meant nesting it into the same class as the call site. You'll see a bunch of places where we do this. Ex `class Foo { static class Factory {` If it is used in multiple classes then yeah it is fine to keep it separate.

extending Call.Base was a suggestion to reduce the amount of code. Generally speaking less untested code the better (ex we don't test the code in tests).